pegasus/sites.v3/code.org/public/blockchain.haml

```diff
@@ -0,0 +1,115 @@
+---
+title: How Blockchain Works
+theme: responsive_full_width
+---
+
+%link{href: "shared/css/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/blockchain-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.bg-neutral-dark{style: "display: block !important"}
+  .wrapper
+    .text-wrapper.col-60
+      %h1{style: "color: white"} How Blockchain Works
```

Comment:
Can this be i18n strings?

Reply:
@mgc1194 yep! I talked about it w/ the team and we can translate the page — however, the lessons won't be translated, but the video series will be.